pkg/simple/client/events/interface.go

```diff
@@ -0,0 +1,44 @@
+package events
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"time"
+)
+
+type Client interface {
+	SearchEvents(filter *Filter, from, size int64, sort string) (*Events, error)
```

Comment:
How to do unit testing without connecting to a real elasticsearch service? Please add more test code to this interface, you can refer to LDAP interface which implements a fake client that can be used to do simple unit testing. This is what we try to achieve, running standalone ks-apiserver or ks-controller-manager without all dependency.

Reply:
Now it is added